grammars/tree-sitter-ssh-server-config/grammar.js

```diff
@@ -43,7 +43,7 @@ module.exports = grammar({
     string: $ => /[^\r\n,"]+/,
 
     _commaSeparatedString: $ => seq($.string, repeat(seq(',', $.string))),
-    _quotedString: $ => seq('\"', $.string, '\"'),
+    quotedString: $ => seq('\"', $.string, '\"'),
```

Comment:
For the parser, you don't care if it's quoted or not, but simply the result is the contents between the quotes (if provided), right?  Can't you define a string as a choice between quoted and not quoted where the quoted is just the contents between the quotes?

Reply:
another option would probably be to update the grammar to not declare space in `extras` but that may have broader impact